crates/llms/Cargo.toml

```diff
@@ -41,8 +41,8 @@ text-splitter = { git = "https://github.com/spiceai/text-splitter.git", rev = "5
 pulldown-cmark = "0.12.1"
 tiktoken-rs = "0.6.0"
 
-mistralrs = { git = "https://github.com/spiceai/mistral.rs", rev = "2978e4039af71007588478be60aa076e2a21437d"}
-mistralrs-core = { git = "https://github.com/spiceai/mistral.rs", rev = "2978e4039af71007588478be60aa076e2a21437d", package = "mistralrs-core" }
+mistralrs = { git = "https://github.com/spiceai/mistral.rs", rev = "048827a2fb9a18820334ec324266568f1203ac86" }
```

Comment:
`2978e4039af71007588478be60aa076e2a21437d` is the current HEAD of the `spiceai` branch - what is `048827a2fb9a18820334ec324266568f1203ac86`?

Reply:
Updated. It was the commit going into `spiceai`.